traitquestdev/index.php

```diff
@@ -16,29 +20,244 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/1.7/jquery.min.js"></script>
 <script type="text/javascript" src="js/contact.js"></script>
+<script type="text/javascript" src="js/bootstrap.js"></script>
 
```

Comment:
add a javascript so that when you click on the menu in the header, it will slide down to the respective section ie. features, testimonial etc